server/bin/pbench-check-tb-age.py

```diff
@@ -143,10 +144,8 @@ def main(options):
     if timediff.days > max_unpacked_age:
         # Finally, make one last check to see if this tar ball
         # directory should be kept regardless of aging out.
-        controller_p = os.path.basename(os.path.dirname(tb_path))
-        if os.path.isfile(
-            os.path.join(incoming_p, controller_p, tb_name, ".__pbench_keep__")
-        ):
+        controller_p = Path(Path(tb_path).parent).name
```

Comment:
Doesn't `Path(tb_path).parent` given you a `Path()` object already?  If so, why not just do, `Path(tb_path).parent.name`?

Reply:
Yaa